tests/test_wps_subset.py

```diff
@@ -0,0 +1,282 @@
+import os
+import sys
+import unittest
+import configparser
+import json
+
+from pywps import Service
+from pywps.tests import client_for
+import numpy.ma as ma
+import netCDF4
+
+try:
+    from tests import test_wps_utils
+except ImportError:
+    import test_wps_utils
+
+
+class TestSubset(unittest.TestCase):
+
+    def setUp(self):
+        self.config = configparser.RawConfigParser()
+        if os.path.isfile('configtests.cfg'):
+            self.config.read('configtests.cfg')
+        else:
```

Comment:
why `if`?

Reply:
@huard That's an Ouranos process. I leave it to you to change it correctly